tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
      * @param {string} permissions
      */
     shareWithUserOrGroup: async function (sharee, shareWithGroup = false, role, permissions) {
+      const util = require('util')
```

Comment:
please put that in the files globals, so we can use it in all functions, then delete the require from all other functions also